clang/test/Driver/sycl-int-header.cpp

```diff
@@ -0,0 +1,86 @@
+/// Check compilation tool steps when using the integration footer
```

Comment:
Test name is `sycl-int-header.cpp` but comments in the test are checking for the footer, but the tests themselves are checking for both.  Can this ben cleaned up?